src/hardware/gus.cpp

```diff
@@ -1251,6 +1249,7 @@ void Gus::RegisterIoHandlers()
 {
 	// Register the IO read addresses
 	assert(read_handlers.size() > 7);
+	using namespace std::placeholders;
```

Comment:
I think it would be better if these were the first line within functions, even before asserts. I don't quite like it when they're in the middle. Could you please address these please, then we're good to go.

Reply:
Moved them all to the start of the method/function.